mesh/settings.py

```diff
@@ -12,6 +12,9 @@
 
 from pathlib import Path
 import os
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
Is there a reason you used `dotenv`? In this file, the `os` library is used to import the `.env` variables, so this is redundant from my perspective.

Reply:
I needed this for some reason at some point, but I just tested it again and it worked fine without it, so I'll remove this!